practice-one/src/layouts/LeftSidebar/LeftSidebar.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import TagLink from "../../components/TagLink/TagLink";
+import { tag_links } from "../../data/tagLinks";
+
+const LeftSidebar = () => {
+  return (
+    <div className="left-sidebar">
+      <ul>
+        {tag_links.map((tag_link, index) => {
+          const { icon, name } = tag_link;
+
+          return (
+            <li key={index}>
```

Comment:
Thanks for your comment, I decided to use uuid for key instead of index 
![image](https://github.com/datdragon777/React-Training/assets/74806826/4acf565d-dfef-45a8-a3ad-b6e61fd79c1c)